examples/medplum-demo-bots/src/billing-bots/cms-1500.test.ts

```diff
@@ -0,0 +1,107 @@
+import { indexSearchParameterBundle, indexStructureDefinitionBundle } from '@medplum/core';
+import { readJson, SEARCH_PARAMETER_BUNDLE_FILES } from '@medplum/definitions';
+import { Bundle, Claim, SearchParameter } from '@medplum/fhirtypes';
+import { MockClient } from '@medplum/mock';
+import { getInsurerInfo, getPatientInfo, getProviderInfo, getReferralInfo, handler } from './cms-1500';
+import { fullAnswer } from './cms-1500-test-data';
+
+const medplum = new MockClient();
+
+describe('CMS 1500 tests', async () => {
+  beforeAll(() => {
+    indexStructureDefinitionBundle(readJson('fhir/r4/profiles-types.json') as Bundle);
+    indexStructureDefinitionBundle(readJson('fhir/r4/profiles-resources.json') as Bundle);
+    for (const filename of SEARCH_PARAMETER_BUNDLE_FILES) {
+      indexSearchParameterBundle(readJson(filename) as Bundle<SearchParameter>);
+    }
+  });
+
+  test('Fullly answered CMS1500', async () => {
+    const result = await medplum.executeBatch(fullAnswer);
+    console.log(result);
+    const claim = (await medplum.searchOne('Claim', {
+      identifier: 'example-claim',
+    })) as Claim;
+
+    const response = await handler(medplum, {
+      bot: { reference: 'Bot/123' },
+      input: claim,
+      secrets: {},
+      contentType: 'application/fhir+json',
+    });
+
+    expect(response).toBeDefined();
+    expectTypeOf(response).toBeString();
+  });
+
+  test('Get patient info', async () => {
+    const patientInfo = getPatientInfo({
+      resourceType: 'Patient',
+      name: [{ given: ['Homer'], family: 'Simpson' }],
+      gender: 'male',
+    });
+
+    expect(patientInfo.patientName).toBe('Homer Simpson');
+    expect(patientInfo.patientSex).toBe('male');
+    expect(patientInfo.patientDob).toBe('');
+    expect(patientInfo.patientAddress).toBe('');
+  });
+
+  test('Insurer is not an organization', async () => {
+    const insurerInfo = getInsurerInfo({
+      resourceType: 'Patient',
+    });
+
+    expect(insurerInfo.serviceNPI).toBe('');
+    expect(insurerInfo.serviceLocation).toBe('');
+    expect(insurerInfo.fedTaxNumber).toBe('');
+    expect(insurerInfo.fedTaxType).toBe('');
+  });
+
+  test('Referrer is not a practitioner or organization', async () => {
+    const patientReferralInfo = getReferralInfo({
+      resourceType: 'Patient',
+    });
+    const organizationReferralInfo = getReferralInfo({
+      resourceType: 'Organization',
+      name: 'Referrer',
+      identifier: [
+        {
+          type: {
+            coding: [{ code: 'NPI' }],
```

Comment:
Does this need to be somethign like https://hl7.org/fhir/sid/us-npi ?  as we show in other places in our documentation?